src/KubernetesClient.Models/StringQuotingEmitter.cs

```diff
@@ -10,7 +10,7 @@ public class StringQuotingEmitter : ChainedEventEmitter
     {
         // Patterns from https://yaml.org/spec/1.2/spec.html#id2804356
```

Comment:
could you please also add `https://yaml.org/type/bool.html`

seems no `on/off` in spec 1.2

Reply:
sure! by any chance, is there any documentation regarding the spec versions that are supported by the k8s yaml parser? shouldn't they be the same for all client libraries + kubectl itself?